internal/services/automation/automation_job_schedule_resource.go

```diff
@@ -279,6 +278,9 @@ func GetJobScheduleFromTFID(ctx context.Context, client *jobschedule.JobSchedule
 	filter := fmt.Sprintf("properties/schedule/name eq '%s' and properties/runbook/name eq '%s'", id.First.ScheduleName, id.Second.RunbookName)
 	jsList, err := client.ListByAutomationAccountComplete(ctx, accountID, jobschedule.ListByAutomationAccountOperationOptions{Filter: &filter})
 	if err != nil {
+		if response.WasNotFound(jsList.LatestHttpResponse) {
+			return nil, nil
+		}
```

Comment:
I tested this and it seems that the response can sometimes be nil causing this to fail

Reply:
> I apologize for not fully testing the app. I believe this issue is caused by an SDK problem and should be resolved in Pandora. I have created a PR to fix it at [hashicorp/pandora#4129](https://github.com/hashicorp/pandora/pull/4129). If possible, could you please review it?

Thanks @wuxu92 - this is now merged 👍